vaadin-grid-cell-click-behavior.html

```diff
@@ -17,32 +17,27 @@
     // yet at the point when tap event is being executed.
     _onClick: function(e) {
       // Prevent item action if cell itself is not focused.
-      if (!this._isFocusable(Polymer.dom(e).localTarget)) {
-        if (this._cellClick) {
-          this._cellClick(e);
+      if (this._cellClick) {
+        if (Polymer.Settings.useNativeShadow) {
```

Comment:
This is the first usage of this setting in `vaadin-grid` and different code for Shadow/Shady DOM doesn't look good for me. Could we somehow get rid of it and use the same logic for both DOM types?